### PR TITLE
feat: format big numbers using `Intl` api when rendered on screen

### PR DESCRIPTION
**Pull request by neuodev**
Format token amount: `100000` ==> `100,000` 

<details><summary>Screenshot</summary>
<p>

Screenshot 2023-04-19 at 12 45 26 AM
Screenshot 2023-04-19 at 12 47 08 AM

</p>
</details>

### CODE REVIEW

**Comment by neuodev** [comment]
[autoscript] ⚠️ `Ahmed/format-nums` doesn't full our branch naming convention.
Should follow `<tag>/<jira-ticket-id>/<short-name>`
<details>
<summary>Details</summary>

- `<tag>`: `fix`, `feat`, `chore`, `refactor`, `test`, ...
- `<jira-ticket-id>`: The issue ID of JIRA all capital letters
- `<short-name>`: A short and descriptive name for the branch
</details>

[](__JIRA_COMMENT_FLAG)

**Comment by neuodev** [comment]
Note: the big diff is because of the formatted file that was not formated

File: packages/yoroi-extension/app/components/wallet/assets/TokenList.js [hunk 1]
```diff
@@ -272,7 +272,9 @@ function TokenItemRow({ avatar, name, id, amount, isTotalAmount }: TokenItemRowP
           <CopyToClipboardText text={id}>{truncateAddressShort(id)}</CopyToClipboardText>
         </Typography>
       }
-      thirdColumn={<Typography fontWeight="500">{amount}</Typography>}
+      thirdColumn={
+        <Typography fontWeight="500">{new Intl.NumberFormat().format(amount)}</Typography>
```
Comment (zuzunker) [inline packages/yoroi-extension/app/components/wallet/assets/TokenList.js]: @neuodev , the number format must be dependent on the selected language. The formatting of the numbers must be matching the selected locale, for now. Later we might think on having it as a separate option, but not right now.

Make sure you resolve the currently selected language and pass it into the `NumberFormat` constructor.

**Review by zuzunker — CHANGES_REQUESTED** [review]
@neuodev , plz check the comment about the locale and also please roll back the formatting changes that are not related to the intended change, it becomes very hard to review and make sure I don't miss any of the changes that are actually important in the PR.

We can have formatting fixes as separate PRs that are not intended to have any logic changes.

File: packages/yoroi-extension/app/components/wallet/assets/TokenList.js (hunk 1 above)
Comment (neuodev) [reply packages/yoroi-extension/app/components/wallet/assets/TokenList.js]: @vsubhuman Fixed! I used our stander way of formating amounts

**Review by zuzunker — APPROVED** [review]

**Pull request merged**